test/options.test.js

```diff
@@ -141,3 +141,34 @@ exports.callingValueOnSomeReturnsValue = function(test) {
     test.deepEqual(4, options.some(4).value());
     test.done();
 };
+
+exports.filteringNoneReturnsNone = function(test) {
+    function equals3(x) {
+        return x === 3;
+    }
+
+    test.deepEqual(options.none.filter(equals3), options.none);
+    test.done();
+};
+
+exports.filteringSomeReturnsIdentityIfPredicateIsTrue = function(test) {
```

Comment:
Agree! Think I'm using that word without knowing what it actually means. I've changed the names of the tests to be more practical similar to what you suggested.